static/src/App.vue

```diff
@@ -22,17 +22,17 @@ export default defineComponent({
       try {
         await PingService.ping();
       } catch (_) {
-        store.jwt = undefined;
```

Comment:
I'm not completely sure, but maybe we could just have written `delete store.jwt` instead of `store.jwt = undefined` here, rendering the additional function unrequired?

Reply:
Doesnt work sadly :(